File: FRI3NDS.Angular4Template.Core/Models/Business/_Admin/_GenericEntity.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FRI3NDS.Angular4Template.Core.Models.Business._Admin
{
    /// <summary>
    /// Обобщенная сущность - набор полей со значениями.
    /// </summary>
    public class _GenericEntity
    {
        /// <summary>
        /// Идентификатор сущности.
        /// </summary>
        public int EntityId { get; set; }

        /// <summary>
        /// Набор обобщенных полей со значениями.
        /// </summary>
        public List<_GenericEntityField> Fields { get; set; }
    }
}

File: FRI3NDS.Angular4Template.Web/app/app/models/business/_GenericEntity.ts
﻿import { _GenericEntityField } from "app/models/business/_GenericEntityField";

export class _GenericEntity {
public constructor(init?: Partial<_GenericEntity>) {
        Object.assign(this, init);
    }
    public entityId: number;
    public fields: _GenericEntityField[];
}

